MinimedKit/HistoryPage.swift

```diff
@@ -30,6 +30,10 @@ public class HistoryPage {
             if let eventType = PumpEventType(rawValue:(pageData[offset] as UInt8)) {
                 let remainingData = pageData.subdata(in: offset..<pageData.count)
                 if let event = eventType.eventType.init(availableData: remainingData, pumpModel: pumpModel) {
+                    if let timestampedPumpEvent = event as? TimestampedPumpEvent {
+                        var timestamp = timestampedPumpEvent.timestamp
+                        timestamp.timeZone = TimeZone(secondsFromGMT: 0)
```

Comment:
This fixes the build but isn't the cleanest implementation. Thoughts @ps2 @loudnate ?